src/library/DIPS.Mobile.UI/Components/BottomSheets/BottomSheetService.cs

```diff
@@ -54,6 +54,29 @@ public static bool IsOpen()
         /// <returns></returns>
         public static partial Task Close(BottomSheet bottomSheet, bool animated = true);
 
+        /// <summary>
+        /// Sets the positioning of the last opened BottomSheet
+        /// </summary>
+        /// <returns>returns false if there is no BottomSheet opened</returns>
+        public static bool TrySetPositionOfLastOpenedBottomSheet(Positioning positioning, out Positioning fromPosition)
```

Comment:
Jeg synes det ble innført litt mer enn det som trengs for å håndtere denne casen på Android i biblioteket vårt.
Som et minimum mener jeg at `TrySetPositionOfLastOpenedBottomSheet` kan fjernes fra denne services og heller legges nærmere Editor og Entry, da dette er veldig spesifikt. Jeg synes også man bør unngå å trenge å hente "siste", selv om det fungerer.

Videre mener jeg at vi ikke trenger å innføre Collapsed, ettersom dette kun er Android spesifikt. Jeg mener det er bedre om Editor og Entry på Android oppdager at de er en del av BottomSheetFragment og heller kjører kode for å sette posisjonen til Collapsed i fragmentet, trenger ikke være en del av Handler da det er Android spesifikt.